lib/rules/space-infix-ops.js

```diff
@@ -48,7 +48,28 @@ module.exports = function(context) {
      * @private
      */
     function report(mainNode, culpritToken) {
-        context.report(mainNode, culpritToken.loc.start, "Infix operators must be spaced.");
+        context.report({
+            node: mainNode,
+            loc: culpritToken.loc.start,
+            message: "Infix operators must be spaced.",
+            fix: function(fixer) {
+                var previousToken = context.getTokensBefore(culpritToken, 1);
+                var afterToken = context.getTokensAfter(culpritToken, 1);
+                var fixString = "";
+
+                if (culpritToken.range[0] - previousToken[0].range[1] === 0) {
+                    fixString = " ";
```

Comment:
I think given a choice between the two, inserting/removing whitespace is the more painful manual operation, so I'm not too worried about triggering `max-len` at this point.